lib/Segments/SegmentSubscribersRepository.php

```diff
@@ -85,6 +86,18 @@ public function getSubscribersCountBySegmentIds(array $segmentIds, string $statu
     return (int)$result;
   }
 
+  public function getDynamicSubscribersCount(DynamicSegmentFilterData $data): int {
```

Comment:
We already have a function for calculating subscribers in dynamic segments. When we display segments we almost always display a count of subscribers next to them. Can't we reuse that? Why do we need a new one? 

This function calculates all subscribers and doesn't filter out deleted, doesn't check status. Is that intentional?

Reply:
I have overlooked this question. I can't reuse the same method because the dynamic segment isn't created or updated in the DB. I know only the filter parameters. I will try to change it a little bit.